navigator/runtime/src/main/java/com/freeletics/mad/navigator/NavEvent.kt

```diff
@@ -16,26 +16,26 @@ public interface NavEvent {
      * Navigates to the given [route].
      */
     public data class NavigateToEvent(
```

Comment:
Could only comment here. 

`import androidx.annotation.IdRes` is unused.

Reply:
[acace8d](https://github.com/freeletics/mad/pull/30/commits/acace8ddd54e09c8690e3b4a1e05246cb0336a0c)